examples/control.ctt

```diff
@@ -0,0 +1,52 @@
+module control where
+
+import prelude
+
+fmap (A B: U) (F: U -> U): (A -> B) -> F A -> F B = undefined
```

Comment:
This looks strange to me. Why not parametrize by a functor instead of having an undefined constant for fmap?

Reply:
This is just not finished, if you accept this, we will plug functor type class to schemes.

Reply:
Just so you know, if you push commits to the branch you made the PR from the changes will appear here. So you should be able to finish this before I merge it. Or do you expect this undefined to be hard to get rid off?

Reply:
Ok, so just plug the functor type class and make one example for List, right?

Reply:
It's not hard but this supposed to be simple example not full featured one, but if you like it I will finish!

Reply:
Yeah, I think the informal policy is that if it should go in the `examples` folder then there should be no `undefined`. The `experiments` folder is more for things that are not finished, however you cannot import things from the `examples` folder with our very primitive import system...

Reply:
Ok acceptance criteria is absence of `undefined`. Got it :-)